doc/plugins/terminal.rst

```diff
@@ -0,0 +1,51 @@
+###############
+Spyder Terminal
+###############
+
+**Spyder-terminal** is a plugin that allows you to have an integrated Terminal inside Spyder.
+
+.. image:: /images/terminal/terminal-standard.png
+   :alt: Spyder Terminal in Spyder
+
+Using the terminal inside Spyder allows you to use the system shell rather than just the IPython console. You can use it to issue commands, interact
```

Comment:
You have to choose if every time you write terminal will be capitalized, as in the previous sentence or without it. I don't have a strong preference about it, so choose the one you like the most.

Reply:
`Using the terminal inside Spyder allows you to use any system shell installed rather than the IPython console.`

Reply:
```suggestion
Spyder-terminal allows you to use any system shell installed in your system (e.g. Bash, Zsh or Powershell) rather than just the IPython console. You can use it to issue commands, interact
```

- This avoids repeating `Using` at the beginning and then `use`.
- It also gives examples of possible shells.